tsconfig.json

```diff
@@ -1,8 +1,11 @@
 {
+  "exclude": ["node_modules"],
```

Comment:
as of now it's easier to just exclude this folder and let TS validate all other *.ts and *tsx files in the repo